docs/usage/.pages

```diff
@@ -54,3 +54,4 @@ nav:
       - 'Known Limitations': 'known-limitations.md'
       - 'Release notes for major versions': 'release-notes-for-major-versions.md'
   - About Us: 'about-us.md'
+  - Contributing to Renovate: 'contributing-to-renovate.md'
```

Comment:
I think "Contributing to Renovate" deserves its own spot in the sidebar. 😉 

But if you want me to move it someplace else, let me know.